decidim-core/app/controllers/decidim/errors_controller.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module Decidim
+  class ErrorsController < ApplicationController
+    authorize_resource :error_pages, class: false
```

Comment:
Mmmm, that's definitely too bare if we want a pretty error page... I tried inheriting from `::ApplicationController` and this is the barest I could get:

```diff
diff --git a/decidim-core/app/controllers/decidim/errors_controller.rb b/decidim-core/app/controllers/decidim/errors_controller.rb
index db803b76..e8e64d2d 100644
--- a/decidim-core/app/controllers/decidim/errors_controller.rb
+++ b/decidim-core/app/controllers/decidim/errors_controller.rb
@@ -1,8 +1,18 @@
 # frozen_string_literal: true
 
 module Decidim
-  class ErrorsController < ApplicationController
-    authorize_resource :error_pages, class: false
+  class ErrorsController < ::ApplicationController
+    include NeedsOrganization
+    include LocaleSwitcher
+
+    helper Decidim::MetaTagsHelper
+    helper Decidim::DecidimFormHelper
+    helper Decidim::LanguageChooserHelper
+    helper Decidim::TranslationsHelper
+    helper Decidim::CookiesHelper
+    helper Decidim::MenuHelper
+
+    layout "layouts/decidim/application"
 
     def not_found
       render status: :not_found
```